book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -189,7 +189,7 @@ You can shut down the cluster with this command at any time:
 cluster_shutdown 127.0.0.1:5050
 ```
 
-You can shut down Giles Sender and Data Receiver by pressing Ctrl-c from their respective shells.
+You can shut down Data Receiver by pressing Ctrl-c from their respective shells.
```

Comment:
Ah, it's not plural anymore, so maybe, "You can shut down Data Receiver by pressing Ctrl-c in the shell"?